tools/istio-iptables/pkg/cmd/root.go

```diff
@@ -373,6 +379,8 @@ func init() {
 
 	rootCmd.Flags().BoolP(constants.DryRun, "n", false, "Do not call any external dependencies like iptables")
 
+	rootCmd.Flags().Bool(constants.TraceLogging, false, "Insert tracing logs for each iptables rules, using the LOG chain.")
```

Comment:
Would be great to use env variables for any new feature - flags have a lot of problems.

Reply:
It is using viper, so has env var + flag. I don't have a strong preference to one or the other but wanted to be consistent with the other configs

Reply:
Sounds good, at some point we should kill viper and just use env - all other feature flags are env.